### PR TITLE
New track review improvemtns

### PR DESCRIPTION
**Pull request by ErikSchierboom**
I missed some review comments.

### CODE REVIEW

File: building/tracks/new/setup-continuous-integration.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ GitHub actions uses the concept of _workflows_, which are scripts that are run a
 Each workflow corresponds to a file in `.github/workflows`.
 Each new track repository comes pre-loaded with three workflows:
 
-- `ci.yml`: this workflow should run the tests for each exercise the track has implemented
+- `test.yml`: this workflow should run the tests for each exercise the track has implemented
```
Comment (SaschaMann) [inline building/tracks/new/setup-continuous-integration.md:11]: This doesn't match any existing files, templates, or existing docs. It's called `ci.yml` everywhere.

Comment (ErikSchierboom) [reply building/tracks/new/setup-continuous-integration.md:11]: Well, this information is just for new tracks and I've renamed it to `test.yml`. `ci.yml` is IMHO not a great name, as _all_ workflows are CI.

Not all tracks have the same naming convention, e.g. C#, Ruby, Go and Python don't.

File: building/tracks/new/select-testing-framework.md [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
-If there are multiple testing frameworks available, we'll need to figure out which one is the best fit.
+If there are multiple testing frameworks available, we will need to figure out which one is the best fit.
 
 ## Considerations
 
 1. Ease of getting started.
 1. Cross-platform support (MacOS, Linux, Windows).
 
-For people who are going to be doing the exercises, it's important to get them up and running as quickly as possible.
+For people who will to be doing the exercises, it's important to get them up and running as quickly as possible.
```
Comment (pertrai1) [inline building/tracks/new/select-testing-framework.md]: should _to_ be removed here?

> For people who will be doing

File: building/tracks/new/setup-continuous-integration.md (hunk 1 above)
Comment (SaschaMann) [reply building/tracks/new/setup-continuous-integration.md:11]: Some workflows are about repo automation and management and the likes, I'd not consider those CI. But I see your point.

I'd suggest you also update the names in other places where `ci.yml` occurs in the docs, like here: https://github.com/exercism/docs/blob/main/building/tracks/ci/workflow-templates.md (probably better in a separate PR, though)

File: building/tracks/new/select-testing-framework.md (hunk 2 above)
Comment (kotp) [inline building/tracks/new/select-testing-framework.md]:
```suggestion
For people who will be doing the exercises, it's important to get them up and running as quickly as possible.
```

File: building/tracks/new/setup-continuous-integration.md (hunk 1 above)
Comment (ErikSchierboom) [reply building/tracks/new/setup-continuous-integration.md:11]: Yeah I'd like that. I'm also simplifying the workflows a bit.

**Review by kotp — APPROVED** [review]

**Pull request merged**